siip.tex

```diff
@@ -305,13 +315,14 @@
           Co-PI Davis will be invited to engage in these video conference as 
           part of action research toward the final 
           As progress is made, lessons learned will be recorded by all 
-          participants in a draft instruction manual. 
+          participants in a draft instruction manual.
+          % TODO: what of Sp18?
```

Comment:
Question was related to what the intention of the interim period is—if there will be more structure to gathering lessons learned prior to the second workshop.  (I imagine there will be conversations, so we don't need anything formal.)